README.md

```diff
@@ -84,42 +100,57 @@ export default class StepPlanet extends Steps {
 }
 ```
 
-The inheritance of the `Steps` class expose following class fields:
+The manager exposed his options to each steps, they can access it with `this.options`.
```

Comment:
_The manager exposes its options on every step. Options can be accessed with `this.options`._

Le manager expose, au présent, ses options à tous les steps. Du coup, pas `ed` à la fin de ` expose`.

Reply:
C'est corrigé